packages/ui-demo/webpack.config.js

```diff
@@ -1,9 +1,23 @@
-// https://github.com/expo/expo/blob/master/packages/expo-yarn-workspaces/README.md
-
-const {createWebpackConfigAsync} = require("expo-yarn-workspaces/webpack");
+const createExpoWebpackConfigAsync = require("@expo/webpack-config");
+const {resolve} = require("path");
 
 module.exports = async function (env, argv) {
-  const config = await createWebpackConfigAsync(env, argv);
+  const config = await createExpoWebpackConfigAsync(env, argv);
```

Comment:
Oh sweet! Was this part of the most recent expo update?

Reply:
I think so! For some reason it was breaking on netlify, no idea why.